app/views/admin/talentlms/edit.haml

```diff
@@ -10,4 +10,5 @@
   = f.error_notification
   = render 'form', f: f
   .form_actions.mt-4
-    = f.submit 'Save Configuration'
\ No newline at end of file
+    = f.submit 'Save Subdomain'
+    =# link_to 'Delete Subdomain', admin_talentlm_path(config), method: :delete, data: {confirm: 'Are you sure you want to delete this configuration? It will no longer be usable.'}, class: 'btn btn-danger'
```

Comment:
Should this be commented out?

Reply:
This comment can be removed, we are deleting configs from the index page.

Cleanup commit: https://github.com/greenriver/hmis-warehouse/pull/4858/commits/fcabe1f421df978bae98d7cf88f84ee6583eb6ba